cypress/integration/url-params.ts

```diff
@@ -0,0 +1,14 @@
+context("Test URL params", () => {
+
+  context("showBurnIndex", () => {
+    it("doesn't render the fire intensity scale when ?showBurnIndex=false", () => {
```

Comment:
What you are actually testing here is when the `showBurnIndex` is omitted, that is the default behavior if not specified.

Reply:
I've fixed the description, thanks.